website/docs/r/cis_domain.html.markdown

```diff
@@ -40,6 +53,10 @@ In addition to all argument reference list, you can access the following attribu
 - `original_name_servers` - (String) The name servers that were used when the domain was first registered with the DNS Registrar.
 - `paused`- (Bool) Indicates if the domain is paused and network traffic bypasses your IBM Cloud Internet Services instance. The default values is **false**.
 - `status` - (String) The status of the domain. Valid values are `active`, `pending`, `initializing`, `moved`, `deleted`, and `deactivated`. After creation, the status remains pending until the DNS Registrar is updated with the CIS name servers, exported in the `name_servers` variable.
+- `type` - (String) The type of domain to be created. Default value is noted to be `full`- for regular domains, & to create a partial domain for CNAME setup, value to be used is `partial`.
```

Comment:
Type is optional varaible it should be added under Argument Reference.
Add the doc changes for datasoruce also